pkg/chat/model.go

```diff
@@ -0,0 +1,37 @@
+package chat
+
+import "time"
+
+type Chat struct {
+	ID			uint				
+	URL	string						//The url of the chatting window
+	DoctorID	uint
+	PatientID	uint
+
+	//primary key: ID
+}
+
+
+type Message struct{
+	ID			uint
+	ChatID		uint
+	Timestamp	Time
+	Type		uint				//Texts, pictures....
+	Content		string
+	PictureURL  string 
```

Comment:
`PictureURL` can be part of content. You can make content an escaped string for that.